<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ponto</class>
 <widget class="QWidget" name="Ponto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Point&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>gcp ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_gcp_id"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeComboBox">
            <item>
             <property name="text">
              <string>Control</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Verification</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Photogrammetric</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditDescription">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Spatial Coordinates</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="SigmaFormDialogButton" name="sigmaDialogButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SigmaFormTypeSelector" name="sigmaSelector"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>E</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="eDoubleSpinBox">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="accelerated">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string/>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>-999999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>999999.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>N</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="nDoubleSpinBox">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="accelerated">
                  <bool>true</bool>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>-9999999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>9999999.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>H</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="hDoubleSpinBox">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="accelerated">
                  <bool>true</bool>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>-9999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>9999.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="SigmaFormContent" name="sigmaContent">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Images Measurements</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="imageMeasurementsTable">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Key</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignHCenter|AlignVCenter|AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Image</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Line</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Column</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SigmaFormTypeSelector</class>
   <extends>QComboBox</extends>
   <header>SigmaForm.h</header>
  </customwidget>
  <customwidget>
   <class>SigmaFormDialogButton</class>
   <extends>QPushButton</extends>
   <header>SigmaForm.h</header>
  </customwidget>
  <customwidget>
   <class>SigmaFormContent</class>
   <extends>QFrame</extends>
   <header>SigmaForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
